pages/checkIn.tsx

```diff
@@ -91,14 +92,16 @@ export default function CheckInPage({
         smOffset={2}
         lgOffset={3}
       >
-        <Button
-          variant="outlined"
-          fullWidth={isMobileView}
-          size="large"
-          sx={{ my: 2 }}
-        >
-          Create new item
-        </Button>
+        <DialogLink href="/items/new">
```

Comment:
this is a fair point and i have no real reason for using `href` other than I like that passing an href gives more 'link-like' feel

Reply:
makes sense, just thought it would reduce errors since you have to specifically add the link into dialogRoutes ... though now that i think of it, it works that way for the href set up too